app/assets/stylesheets/building_blocks.scss

```diff
@@ -0,0 +1,75 @@
+html,body {
+  background: #FAFAFA;
+}
+
+.card {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Properties should be ordered background, border-radius, box-shadow, color, font-family, overflow